src/middle-end/counter/index.js

```diff
@@ -1,5 +1,10 @@
-module.exports = {
-    reducer: require('./reducer'),
-    actions: require('./actions'),
-    selectors: require('./selectors')
+module.exports = (m, options) => {
+
+    return {
+        reducer: require('./reducer'),
+        actions: require('./actions')(m),
```

Comment:
Not because it's needed, just to create less variation.
```suggestion
        actions: require('./actions')(m, options),
```

Reply:
ah, werd, I waffled on this, then did the dugg. thanks for noting this, fixing up now!